compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -304,6 +303,13 @@ private void defineConstructs(BLangPackage pkgNode, SymbolEnv pkgEnv) {
         pkgNode.typeDefinitions.forEach(typDef -> typDefs.add(typDef));
         defineTypeNodes(typDefs, pkgEnv);
 
+        for (BLangSimpleVariable variable : pkgNode.globalVars) {
+            if (variable.expr != null
```

Comment:
Shall we move all the checks to same line, and get variable.expr related checks closer
`variable.expr != null && variable.expr.getKind() == NodeKind.LAMBDA && variable.isDeclaredWithVar`

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/19976/commits/7432217a84fd35c7ae7682d1e9173e9416f9ccdc